src/api/app/views/webui/request/beta_show_tabs/_overview.html.haml

```diff
@@ -1,6 +1,48 @@
-%h4.list-group.mb-4 Comments & Request History
-.comments-list{ data: { comment_counter: local_assigns[:comment_counter_id] } }
-  = render BsRequestActivityTimelineComponent.new(bs_request: bs_request)
+.row
+  .col-md-12#overview
+    .mb-4#description-text
+      - if bs_request.description.present?
+        = render partial: 'webui/shared/collapsible_text', locals: { text: bs_request.description }
+      - else
+        %i No description set
+    .row
+      .col-md-4.order-md-2.order-sm-1.mb-4#side-links
+        %ul.list-unstyled
+          - if bs_request.accept_at.present?
+            %li
+              %i.fas.fa-exclamation-circle.text-danger
+              - if BsRequest::FINAL_REQUEST_STATES.include?(bs_request.state)
+                Auto-accept was set to
+                %span.fuzzy-time{ title: "#{l bs_request.accept_at}" }
+                = succeed '.' do
+                  = l bs_request.accept_at, format: :only_date
+              - elsif bs_request.accept_at.past?
+                This request will be automatically accepted when it enters the 'new' state.
+              - else
+                This request will be automatically accepted in #{time_ago_in_words(bs_request.accept_at)}.
+          - open_reviews = bs_request.reviews.opened.for_non_staging_projects
+          - if bs_request.accepted_reviews.present? || bs_request.declined_reviews.present? || open_reviews.present?
+            .row
+              .col-auto.pr-0
+                %h4
+                  Reviews
+              = render partial: 'webui/request/beta_show_tabs/review_summary',
+                      locals: { reviews: bs_request.reviews.accepted.for_non_staging_projects, key: :accepted_reviews }
 
-  .comment_new.mt-3
-    = render partial: 'webui/comment/new', locals: { commentable: bs_request }
+              = render partial: 'webui/request/beta_show_tabs/review_summary',
+                      locals: { reviews: bs_request.reviews.declined.for_non_staging_projects, key: :declined_reviews }
+
+              = render partial: 'webui/request/beta_show_tabs/review_summary', locals: { reviews: open_reviews.for_non_staging_projects, key: :open_reviews }
+
+          - bs_request.reviews.opened.for_staging_projects.each do |review|
+            %li
```

Comment:
This looks a bit weird now.

![Screenshot from 2022-08-09 15-37-41](https://user-images.githubusercontent.com/514785/183665967-52604bb3-bd90-4bdc-af6b-f71fa82f2ec3.png)

The HTML structure also is weird.

![Screenshot from 2022-08-09 15-53-00](https://user-images.githubusercontent.com/514785/183666973-79cc2dee-28ab-4bfc-8abf-fb3c44cf5053.png)


I guess you want to drop the outer `<ul>` and style this only with rows/cols?

Reply:
We removed the `<ul>` we didn't need and simplified the structure of the HTML.

This is the result so far. We know the lines regarding staging projects and automatic acceptance can be improved. We'll postpone it. We created two cards in the backlog for that.

- https://trello.com/c/B0V5onJP/2675-find-a-better-way-to-display-the-staged-project-reviews
- https://trello.com/c/NBfrMFR0/2676-find-a-better-way-to-display-the-messages-about-automatic-acceptance

@hennevogel  please take another look.

![Screenshot 2022-08-09 at 17-08-43 Open Build Service](https://user-images.githubusercontent.com/2581944/183692340-1fe91d5c-2538-42d5-9254-d15ae5023ea9.png)